bot/mc.go

```diff
@@ -75,7 +75,12 @@ func execDockerCompose(action string) error {
 	if err != nil {
 		return fmt.Errorf("ssh error: %w", err)
 	}
-	defer client.Close()
+
+	defer func() {
+		if cerr := client.Close(); cerr != nil && err == nil {
+			err = cerr
+		}
+	}()
```

Comment:
```suggestion
	defer func() {
		if err := client.Close(); err != nil {
			log.Printf("Error! %w", err)
		}
	}()
```

You should think more about why this code looks wrong, assigning to `err` does nothing here since we're in a defer block (that will get called _after_ the function fully completes).